insights/specs/datasources/ipcs.py

```diff
@@ -0,0 +1,47 @@
+"""
+Custom datasources to get the semid of all the inter-processes.
+"""
+
+from insights.core.context import HostContext
+from insights.core.spec_factory import simple_command
+from insights.core.plugins import datasource
+from insights.specs import Specs
+from insights.core.dr import SkipComponent
+
+
+class LocalSpecs(Specs):
+    """ Local specs used only by semid datasources """
+
+    ipcs_s_cmd = simple_command("/usr/bin/ipcs -s")
```

Comment:
There's no need to create this, just reference the spec `insights.specs.Specs.ipcs_s` instead.

Reply:
@bfahr, sure, updated, thanks.